### PR TITLE
Update Rosetta Readme to the latest specs

### PR DESCRIPTION
**Pull request by kaozenn**

### CODE REVIEW

**Comment by kaozenn** [comment]
!ci-build-me

File: src/app/rosetta/README.md [hunk 1]
```diff
@@ -6,13 +6,15 @@ Implementation of the [Rosetta API](https://www.rosetta-api.org/) for Mina.
 
 Checkout the "master" branch of the mina repository, ensure your Docker configuration has a large amount of RAM (at least 12GB, recommended 16GB) and then run the following:
 
+### Release v1.3.0 (Mainnet and Devnet)
```
Comment (simisimis) [inline src/app/rosetta/README.md:9]: not a blocker but on documentation latest stable release is 1.3.1.2

**Review by simisimis — APPROVED** [review]

**Comment by robinbb** [comment]
Good work on the attention to details in this PR, @simisimis and @kaozenn !

File: src/app/rosetta/README.md (hunk 1 above)
Comment (kaozenn) [reply src/app/rosetta/README.md]: Good catch @simisimis. I have updated the rest of the Readme accordingly.

**Comment by kaozenn** [comment]
!ci-build-me

**Comment by kaozenn** [comment]
!ci-build-me

**Pull request merged**